### ISSUE TITLE
Medusa can no longer snatch NZBs?

### ISSUE THREAD

**Issue by AshleyAshes** [pymedusa/Medusa#3525]
### Before submitting your issue:

Enable debug logging in Medusa settings, reproduce the error (be sure to disable after the bug is fixed)

**Branch/Commit: Master, b6f18141379fcb2f61bcfe7beaeebca442892aef**
**OS: Windows 10 Pro**
**What you did: I did update master before it happened**
**What happened: NZBs stopped snatching**
**What you expected:  ...NZBs to snatch?**
**Logs:**

@p0psicles
```
2017-12-27 01:05:36 ERROR    SEARCHQUEUE-DAILY-SEARCH :: [b6f1814] DailySearchQueueItem Exception, error: 'NZBSearchResult' object has no attribute 'resultType'
Traceback (most recent call last):
  File "C:\Medusa\Medusa\medusa\search\queue.py", line 288, in run
    self.success = snatch_episode(result)
  File "C:\Medusa\Medusa\medusa\search\core.py", line 138, in snatch_episode
    result_downloaded = _download_result(result)
  File "C:\Medusa\Medusa\medusa\search\core.py", line 67, in _download_result
    if result.resultType == u'nzb':
AttributeError: 'NZBSearchResult' object has no attribute 'resultType'



2017-12-27 01:38:49 ERROR    SNATCHQUEUE-MANUALSNATCH-332987 :: [b6f1814] Manual snatch failed!. For result: [HorribleSubs].Animegataris.-.12.[1080p]
Traceback (most recent call last):
  File "C:\Medusa\Medusa\medusa\search\queue.py", line 487, in run
    self.success = snatch_episode(result)
  File "C:\Medusa\Medusa\medusa\search\core.py", line 138, in snatch_episode
    result_downloaded = _download_result(result)
  File "C:\Medusa\Medusa\medusa\search\core.py", line 67, in _download_result
    if result.resultType == u'nzb':
AttributeError: 'NZBSearchResult' object has no attribute 'resultType'


@p0psicles I get this with any kind of  NZB snatch now.  I don't see how I've done anything to cause it.

**Comment by p0psicles** [comment]
We'll throw out a hotfix soon.
But just curious, which provider are you using?

**Comment by p0psicles** [comment]
@AshleyAshes any way you can checkout this branch: `hotfix/fix-resultType-bug` and test the fix?

**Comment by p0psicles** [comment]
Should be fixed if you update master to release 0.1.22

**Comment by AshleyAshes** [comment]
Seems you fixed it while I was sleep and before I could reply.  I updated master and it works great.  You're always super fast when I report issues, you're awesome. :)
